app/models/vm_or_template/operations/configuration.rb

```diff
@@ -73,8 +73,9 @@ def disconnect_floppies
 
   def raw_add_disk(disk_name, disk_size_mb, options = {})
     raise _("VM has no EMS, unable to add disk") unless ext_management_system
-    if options[:datastore]
-      datastore = Storage.find_by(:name => options[:datastore])
+    if options[:datastore] || options[:datastore_id]
```

Comment:
Similar logic was implemented on https://github.com/ManageIQ/manageiq-providers-ovirt/pull/37/files#diff-17198062ae8d3e8788a4f5db1eb17ccdR53

Should it be pulled to a common place and be reused ? If so, any suggestions for such place in the code ?